src/test/java/edu/cornell/kfs/sys/service/impl/fixture/AwsSecretePojoBasic.java

```diff
@@ -0,0 +1,58 @@
+package edu.cornell.kfs.sys.service.impl.fixture;
+
+import java.util.Date;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+import com.google.gson.annotations.JsonAdapter;
+
+import edu.cornell.kfs.sys.jsonadapters.JsonDateSerializer;
+
+public class AwsSecretePojoBasic {
+    
+    private String static_string;
+    private String changeable_string;
+    private int number_test;
+    private boolean boolean_test;
+    
+    @JsonAdapter(JsonDateSerializer.class)
+    private Date update_date;
+    
+    public String getChangeable_string() {
+        return changeable_string;
+    }
+
+    public void setChangeable_string(String changeable_string) {
+        this.changeable_string = changeable_string;
+    }
+
+    public int getNumber_test() {
```

Comment:
Is this numeric field meant to be static? I don't see a setter for it.

Reply:
It was my proof of concept to show that we can have pojos for key value pairs where we don't intend to have KFS update the value.  An example would be the web service credential values for concur that are not updatable, like the user name and password

Reply:
Ok, thanks for clarifying.